scripts/build-site.sh

```diff
@@ -33,7 +38,7 @@ cp -f COVID-Hospitalization/Archiv/`ls COVID-Hospitalization/Archiv | tail -1` s
 echo BUILD: Create [city]-cases.csv files
 # create the [city]-cases.csv files
 python3 --version
-python3 scripts/rki-update.py rki.csv
```

Comment:
Isn't this running from the wrong folder? The script is run from the root folder, not from `scripts`